go.mod

```diff
@@ -21,6 +21,8 @@ require (
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.3
+	github.com/tidwall/gjson v1.17.1
+	github.com/tidwall/sjson v1.2.5
```

Comment:
The last commit of `sjson` happened 2 years ago, do we have any other alternatives?

Reply:
We need json library to read and write json string directly, skip serialization.

Reply:
We need json library to read and write json string directly, skip serialization.

As for **get json string** function, here are several alternative, like:
* https://github.com/buger/jsonparser
* https://github.com/Jeffail/gabs
* https://github.com/bitly/go-simplejson
* https://github.com/antonholmquist/jason

However, only  `go-simplejson` is closer to the last update time, others are also several years no update. But they all have lower star number than `gjson`. 

As for **set json string** function, I haven't found any other alternatives yet.

Reply:
I talked to some users about their practices regarding inactive dependencies, they generally do not allow to use dependencies that have been inactive for more than 3 years.

So, I think we can use the `github.com/tidwall/sjson` for now, and if still no new releases next year, we can replace it with go-simplejson.

What do you say?

Reply:
makes perfect sense !